package.json

```diff
@@ -79,24 +79,28 @@
   "dependencies": {
     "3box": "1.17.1",
     "@burner-wallet/burner-connect-provider": "^0.1.1",
-    "@daostack/client": "0.2.67",
-    "@dorgtech/daocreator-ui": "^1.0.8",
+    "@dorgtech/arc.js": "2.0.0-experimental.32",
+    "@dorgtech/daocreator-ui-experimental": "1.1.2",
     "@fortawesome/fontawesome-svg-core": "^1.2.10",
     "@fortawesome/free-brands-svg-icons": "^5.6.1",
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@portis/web3": "^2.0.0-beta.43",
     "@sentry/browser": "^5.0.8",
     "@toruslabs/torus-embed": "^1.3.0",
     "@walletconnect/web3-provider": "^1.0.0-beta.37",
+    "apollo-link-retry": "^2.2.16",
```

Comment:
this should not be part of arc.js ?

Reply:
yep, you're correct it is, this can be removed

Reply:
Actually, apologies, this is required since Alchemy imports this package in arc.ts and settings.ts. I think it's best not to assume arc.js will include this package, and instead import it ourselves.